week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,17 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  return ((price + foreignTransactions(price)) * 1.4).toFixed(2); //price in USD
+}
+function convertToBRL(price) {
+  return ((price + foreignTransactions(price)) * 4.89).toFixed(2); // price in BRL
+}
+
+function foreignTransactions(price) {
+  // transaction fee
+  return price * 0.01;
+}
```

Comment:
Well done!
Can you think of a way to simplify this code? For example, in the conversion functions, you always use the `price` variable and add the fee to it separately. Could be done in one function, no?

Reply:
Do you mean ,neglect foreignTransaction function?